src/EditorFeatures/Test/Preview/PreviewWorkspaceTests.cs

```diff
@@ -236,7 +224,7 @@ public void TestPreviewDiagnostic()
                 {
                     var foregroundService = workspace.GetService<IForegroundNotificationService>();
 
-                    var listenerProvider = new AsynchronousOperationListenerProvider();
+                    var listenerProvider = workspace.ExportProvider.GetExportedValue<AsynchronousOperationListenerProvider>();
```

Comment:
again, to test like this, it must have its own ExportProvider rather than sharing common one with other test. ExportProvider.Create....